packages/echo-core/src/__tests__/actions/coreActions/extensions.test.ts

```diff
@@ -0,0 +1,151 @@
+import { registerExtension, registerMultipleExtensions } from '../../../actions/coreActions/extensions';
+import { readState } from '../../../actions/coreActions/globalActions';
+import { getCoreContext } from '../../../state/globalState';
+import { ExtensionRegistration } from './../../../types/registry/extension.types';
+
+describe('extensions', () => {
+    describe('registerExtension()', () => {
+        it('should register and unregister a given extension into the state', () => {
+            // when
+            const unregisterExtension = registerExtension(extension);
```

Comment:
intellisense is not able to figure out the return type here, it just says : VoidFunction
Are we able to fix this? or is it a limitation of the intellisense?

Reply:
What would you expect to see here? 🤔 An unregister function is really a void function, without any parameters, without any return type.